Documentation/Grafana-Dashboards/ContainerLogV2.json

```diff
@@ -0,0 +1,2533 @@
+{
+    "annotations": {
+      "list": [
```

Comment:
1. Lets update the proper color code for log level tiles. 
2.  Fix the text "Log/min by LogLevel" to "Logs/min by LogLevel"
3. Move the Logs/min by LogLevel to the top and below pie chart